Emby.Server.Implementations/Library/LiveStreamHelper.cs

```diff
@@ -48,20 +48,25 @@ public async Task AddMediaInfoWithProbe(MediaSourceInfo mediaSource, bool isAudi
 
             if (!string.IsNullOrEmpty(cacheKey))
             {
-                FileStream jsonStream = AsyncFile.OpenRead(cacheFilePath);
                 try
                 {
-                    mediaInfo = await JsonSerializer.DeserializeAsync<MediaInfo>(jsonStream, _jsonOptions, cancellationToken).ConfigureAwait(false);
+                    FileStream jsonStream = AsyncFile.OpenRead(cacheFilePath);
 
-                    // _logger.LogDebug("Found cached media info");
-                }
-                catch (Exception ex)
-                {
-                    _logger.LogError(ex, "Error deserializing mediainfo cache");
+                    try
+                    {
+                        mediaInfo = await JsonSerializer.DeserializeAsync<MediaInfo>(jsonStream, _jsonOptions, cancellationToken).ConfigureAwait(false);
+                        // _logger.LogDebug("Found cached media info");
+                    }
+                    catch (Exception ex)
+                    {
+                        _logger.LogError(ex, "Error deserializing mediainfo cache");
+                    }
+
+                    await jsonStream.DisposeAsync().ConfigureAwait(false);
                 }
-                finally
+                catch
                 {
-                    await jsonStream.DisposeAsync().ConfigureAwait(false);
+                    _logger.LogError("Could not open cached media info");
```

Comment:
Might as well log the exception here

Reply:
This error is quite common and nothing to be worried about as it will then create the file. I can add the exception if you think that would be better, i was just worried about flooding the console.

Reply:
It's not really an error then. It should be debug. Error level is when something should be looked at.

Reply:
Okay i have added this, please take a look in the latest commit and tell me what you think :)